website/docs/r/vapp_vm.html.markdown

```diff
@@ -288,14 +290,44 @@ Changes are ignored on update. This part isn't reread on refresh. To manage inte
+vApp template. When you customize your guest operating system, you can create and deploy multiple unique virtual
+machines based on the same vApp template without machine name or network conflicts.
+
+When you configure a vApp template with the prerequisites for guest customization and add a virtual machine to a vApp
+based on that template, vCloud Director creates a package with guest customization tools. When you deploy and power on
+the virtual machine for the first time, vCloud Director copies the package, runs the tools, and deletes the package from
+the virtual machine.
+
+~> **Note:** The settings below work so that all values are inherited from template and only the specified fields are
+overridden with exception being `force` field which works like a flag.
+
 * `force` (Optional) **Warning.** `true` value will cause the VM to reboot on every `apply` operation.
 This field works as a flag and triggers force customization when `true` during an update 
 (`terraform apply`) every time. It never complains about a change in statefile. Can be used when guest customization
 is needed after VM configuration (e.g. NIC change, customization options change, etc.) and then set back to `false`.
 **Note.** It will not have effect when `power_on` field is set to `false`. See [example workflow below](#example-forced-customization-workflow).
+* `enabled` (Optional; *v2.7+*) `true` will enable guest customization which may occur on first boot or if the `force` flag is used.
+This option should be selected for **Power on and Force re-customization to work**. For backwards compatibility it is
+enabled by default when deprecated field `initscript` is used.
+* `change_sid` (Optional; *v2.7+*) Allows to change SID (security identifier). Only applicable for Windows operating systems.
+* `allow_local_admin_password` (Optional; *v2.7+*) Allow local administrator password.
+* `must_change_password_on_first_login` (Optional; *v2.7+*) Require Administrator to change password on first login.
+* `auto_generate_password` (Optional; *v2.7+*) Auto generate password.
+* `admin_password` (Optional; *v2.7+*) Manually specify Administrator password.
+* `number_of_auto_logons` (Optional; *v2.7+*) Number of times to log on automatically. `0` means disabled.
+* `join_domain` (Optional; *v2.7+*) Enable this VM to join a domain.
+* `join_org_domain` (Optional; *v2.7+*) Set to `true` to use organization's domain.
+* `join_domain_name` (Optional; *v2.7+*) Set the domain name to override organization's domain name.
+* `join_domain_user` (Optional; *v2.7+*) User to be used for domain join.
+* `join_domain_password` (Optional; *v2.7+*) Password to be used for domain join.
+* `join_domain_account_ou` (Optional; *v2.7+*) Organizational unit to be used for domain join.
+* `initscript` (Optional; *v2.7+*) Provide initscript to be executed when customization is applied.
 
 ## Example forced customization workflow
```

Comment:
```suggestion
## Example of a Forced Customization Workflow
```

Also, nit: the `// ...` in the examples below are out of indent and missing new line after them.

<img width="573" alt="Screenshot 2020-03-03 at 16 19 20" src="https://user-images.githubusercontent.com/11330689/75784468-e51cae00-5d6a-11ea-89df-427b051cb4f3.png">

Reply:
Changed  from `//` to `#` (the real HCL comment symbol).